src/cache/mod.rs

```diff
@@ -991,7 +991,9 @@ mod test {
                 application: None,
                 message_reference: None,
                 flags: None,
+                stickers: vec![],
                 referenced_message: None,
+                _nonexhaustive: (),
```

Comment:
Please remove this; serenity was moved to `#[non_exhaustive]` a few days ago, meaning this is no longer needed.